tests/Application/symfony.lock

```diff
@@ -1,890 +0,0 @@
-{
```

Comment:
I'm not 100% sure that symlinking this file is a good idea. What if we `composer update` the project?
How will it react?